sympy/matrices/expressions/tests/test_matexpr.py

```diff
@@ -76,6 +76,40 @@ def test_ZeroMatrix_doit():
     assert isinstance(Znn.doit().rows, Mul)
 
 
+def test_OneMatrix():
+    A = MatrixSymbol('A', n, m)
+    a = MatrixSymbol('a', n, 1)
+    U = OneMatrix(n, m)
+
+    assert U.shape == (n, m)
+
+    assert isinstance(A + U, Add)
+    # assert a*OneMatrix(n, 1) == a
+    # assert OneMatrix(n, 1)*a == a
```

Comment:
Would these commented-out tests should be made more meaningful?

And I don't think that the matrix multiplication can be defined here, unless n is 1.

Reply:
These are wrong. I shall remove them.